cardano-node/src/Cardano/CLI/Run.hs

```diff
@@ -19,6 +20,7 @@
 {-# OPTIONS_GHC -Wno-all-missed-specialisations #-}
 {-# OPTIONS_GHC -Wno-simplifiable-class-constraints #-}
 {-# OPTIONS_GHC -Wno-partial-fields #-}
+{-# OPTIONS_GHC -Wno-incomplete-uni-patterns #-}
```

Comment:
Just saw this. Please handle the other cases explicitly.

Reply:
I found a way to convince GHC that the impossible branch is indeed impossible (removed a type variable + a type equality that pinned a branch in the `ProtocolInfo` GADT), so this is now unnecessary.

Done.